keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -169,6 +194,151 @@ import (
 )
 ```
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: DisableCloudProviders
+    - Components depending on the feature gate: kubelet, kube-apiserver, kube-controller-manager
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: DisableKubeletCloudCredentialProvider
+    - Components depending on the feature gate: kubelet
+
+* **Does enabling the feature change any default behavior?**
+  Yes, enabling this feature will disable all capabilities enabled when --cloud-provider is set.
```

Comment:
Minor point but it disabled the capabilities within the core components. If the appropriate extension points (CCM, Credential Provider, APIServer Network Proxy) are in use, the capabilities should remain functional.

Reply:
Added that, thanks